tests/dummy/config/environment.js

```diff
@@ -18,7 +18,8 @@ module.exports = function(environment) {
       // when it is created
     },
 
-    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/'
+    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/',
+    bootstrapVersion: process.env.BOOTSTRAPVERSION || 3
```

Comment:
But this is the config of the addon's dummy app, users should not see this. And unless they don't add the same logic into their own config file, they will not be able use that. It's a bit different for the `index.js`, as this also affects the user's app. But usually they won't look into that file really...